cmd/juju/status/status_internal_test.go

```diff
@@ -4708,6 +4708,84 @@ foo/1  active    running     10.0.0.1  80/TCP
 `[1:])
 }
 
+func (s *StatusSuite) TestFormatTabularStatusNotes(c *gc.C) {
+	fStatus := formattedStatus{
+		Model: modelStatus{
+			Type: "caas",
+		},
+		Applications: map[string]applicationStatus{
+			"foo": {
+				Address: "54.32.1.2",
+				StatusInfo: statusInfoContents{
+					Message: "Error: ImagePullBackOff",
+				},
+				Units: map[string]unitStatus{
+					"foo/0": {
+						Address:     "10.0.0.1",
+						OpenedPorts: []string{"80/TCP"},
+						JujuStatusInfo: statusInfoContents{
+							Current: status.Running,
```

Comment:
Unit won't be running if there's an error in the operator startup. It will be "allocating"/"waiting for container" or whatever.
Besides that, "running" isn't a valid agent status